book/src/nostr/06-nip06.md

```diff
@@ -1 +1,71 @@
-# NIP-06
+## NIP-06
+
+In accordance with the [Bitcoin Improvement Proposal 0039 (BIP-39)](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) we can derive Nostr keys using seed phrases as a source of entropy. This is handled by the [FromMnemonic](https://docs.rs/nostr/latest/nostr/nips/nip06/trait.FromMnemonic.html) Trait and its associated methods.
+
+The default functionality is to generate a single key-pair at the derivation path `0`. However, it is also possible to perform more advanced derivations by incrementing the `account`, enabling generation of many sets of keys from a single seed. More granular derivation path control (over `account`, `type` and `index`) exists within the `from_mnemonic_advanced` method. 
+
+For more examples of key generation please refer back to the [Keys section](https://rust-nostr.org/nostr/03-keys.html) of this book.
```

Comment:
For any link to sections of the book, not use the domain. Intead use the path to file so, for example, `03-keys.md`.

```md
[Keys section](03-keys.md)
```